app/src/main/res/values/strings.xml

```diff
@@ -818,4 +818,6 @@ Upload your first media by tapping on the add button.</string>
   </plurals>
   <string name="multiple_files_depiction">Please remember that all images in a multi-upload get the same categories and depictions. If the images do not share depictions and categories, please perform several separate uploads.</string>
   <string name="multiple_files_depiction_header">Note about multi-uploads</string>
+  <string name="nearby_wikitalk">wikitalk</string>
```

Comment:
This is the tooltip right? How about "Report a problem about this item to Wikidata"?